common/src/main/java/org/spine3/samples/lobby/common/util/aggregate/LobbyAggregate.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2015, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.samples.lobby.common.util.aggregate;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.protobuf.Message;
+import org.spine3.base.CommandContext;
+import org.spine3.base.Event;
+import org.spine3.samples.lobby.common.ImportEvents;
+import org.spine3.server.aggregate.Aggregate;
+
+import java.util.List;
+
+/**
+ * @author Dmytro Dashenkov
+ */
+@SuppressWarnings("AbstractClassWithoutAbstractMethods")
+public abstract class LobbyAggregate<I, S extends Message, B extends Message.Builder> extends Aggregate<I, S, B> {
```

Comment:
Please add class-level JavaDocs. This is a requirement for all classes in Spine.